### ISSUE TITLE
Too much output when using quad ⎕

### ISSUE THREAD

**Issue by rikedyp** [Dyalog/PracticeProblems#7]
**To Reproduce**
1. Go to  https://problems.tryapl.org/
2. Scroll down to sample problem input
4. Try `{⎕←'ok'}¨`

We should probably suppress ⎕ and ⍞ output if possible, but it's low priority (anyone who knows what ⎕ and ⍞ do should be able to figure out what's going on)

**Comment by abrudz** [comment]
Will go away when we switch to TryAPL-based execution

**Comment by awagga** [comment]
With ATO-based execution and the "user feedback" improvement this case will output nothing. `rx.exec(...)` returns null for ⎕←, ⎕off, etc.

**Comment by abrudz** [comment]
Since we've switched to ATO backend, the issue is now (ironically) too *little* output.

**Comment by awagga** [comment]
Does '...should have returned 3 with ('COOLAPL') as right argument.' sound appropriate for early halting expressions? (To clarify these expressions return un-expected output, or often errors - which we ignore. I think it's ok, though naive, to simply detect the null and ignore the details)

**Comment by abrudz** [comment]
Yes, I think so. Though technically `{⎕←'huh' ⋄ +/⍵∊'AEIOU'}` should/could(?) pass…

**Comment by awagga** [comment]
That's a good point, though handling that would require changes to the framework, or possibly subsequent requests for round-tripping purposes.
